src/vellum/workflows/nodes/displayable/bases/api_node/node.py

```diff
@@ -43,6 +43,34 @@ class Outputs(BaseOutputs):
         status_code: int
         text: str
 
+    def _normalize_http_method(self, method: Union[APIRequestMethod, str]) -> APIRequestMethod:
```

Comment:
[nit] Both `_vellum_execute_api` and `_local_execute_api` eventually just convert back to a string with `method.value` so I could actually see normalizing the opposite direction